redhat/frr.spec.in

```diff
@@ -566,7 +566,8 @@ zebra_spec_add_service fabricd	2618/tcp "Fabricd vty"
 # Create dummy config file if they don't exist so basic functions can be used.
 if [ ! -e %{configdir}/zebra.conf ]; then
     # per daemon configs exist
-    mv %{configdir}/frr.conf.template %{configdir}/frr.conf
+    test ! -e %{configdir}/frr.conf && \
+        mv %{configdir}/frr.conf.template %{configdir}/frr.conf
```

Comment:
Can you please just use `if`  just to be consistent with the syntax used two lines above.